libs/template/application/server/src/server.prod.ts

```diff
@@ -31,7 +31,6 @@ export async function createProdSever(
 
       res.status(200).end(html);
     } catch (e) {
-      console.error(e);
```

Comment:
Do we have to remove it from here?
This error appears in Netlify function logs and might be useful for debugging

Reply:
Moved back, thanks.